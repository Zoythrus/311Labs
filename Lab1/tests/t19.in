stat
20 20 100 intersection
61
43
96
38
20
56
27
78
70
42
100
4
45
93
44
87
77
91
63
90
8
0
42
46
18
19
82
85
40
71
64
35
68
12
77
94
65
44
95
24
