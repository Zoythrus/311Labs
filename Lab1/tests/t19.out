42
44
77
